com.microsoft.mrtk.graphicstools.unity/Editor/Experimental/AmbientOcclusion/README.md

```diff
@@ -0,0 +1,45 @@
+# Ambient occlusion
```

Comment:
Noting for myself to eventually move this to: https://learn.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-graphicstools/